modules/ROOT/pages/install-odatav2-tooling.adoc

```diff
@@ -0,0 +1,24 @@
+= Installing OData V2 Tooling
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+To install APIkit for OData v2 plugin in Anypoint Studio:
+
+. Select *Help* > *Install New Software* and click *Add*.
```

Comment:
```suggestion
. Open *Help* > *Install New Software* and click *Add*.
```
That or "Go to". I think these are more accurate than "Select" in this case.

Reply:
OK! Thanks!